src/borg/archiver.py

```diff
@@ -1363,10 +1363,11 @@ def _list_repository(self, args, manifest, write):
         output_data = []
 
         for archive_info in manifest.archives.list_considering(args):
+            archive = Archive(repository, key, manifest, archive_info.name)
```

Comment:
this is a bit more expensive now as it load the Archive metadata, not just the manifest.

only matters for huge numbers of archives and in that case I guess there are bigger performance problems elsewhere.